stories/Announcer.stories.js

```diff
@@ -35,4 +36,4 @@ class AnnouncerTest extends Component {
     }
 }
 
-export default AnnouncerTest;
```

Comment:
Do we still want to run tests in this repo?

We still have the `test` script defined in `package.json` (in fact that seems to be the only reason to include `react-scripts` in this repo's dependencies), but the only test I see for now is
https://github.com/cyverse-de/de-components/blob/master/src/App.test.js

It might still be a good idea to have at least simple tests running while developing these components (like those in the DE UI's `react-components`).

We could keep these default exports for Jest tests, or we can add them back in a future PR when the tests are added too.

Reply:
Agreed. we need to have some tests. Will try to add some. 👍